src/components/basic/SelectList/SelectList.stories.tsx

```diff
@@ -79,6 +79,7 @@ SelectList.args = {
   clearText: 'clear',
   noOptionsText: 'No Options',
   popperHeight: 0, // 0 = auto size
+  defaultValue: countries[0],
```

Comment:
@lavanya-bmw what does this default value "countries" stand for? I can not connect it to the task ticket

Reply:
@jjeroch this is stories file which displays a default value to the items(countries) from line 70. This change is to see the output in shared components.